### PR TITLE
Release/iqv 1.5.3 prewhitelabel

### PR DESCRIPTION
**Pull request by gzach**
This is the HyBid code base that the new iqv v1.5.3 release is based on.
It is based on development + recent pull requests on HyBid + a few last minute changes. (see commit log below)

There are no IQV specific code changes in this branch. All whitelabelling is done by Scripts/iqv.sh
It should be safe to merge this back into development.

### CODE REVIEW

File: PubnativeLite/PubnativeLite/Ad Request/PNLiteAdFactory.m [hunk 1]
```diff
@@ -49,9 +49,8 @@ - (PNLiteAdRequestModel *)createAdRequestWithZoneID:(NSString *)zoneID
     [self setIDFA:adRequestModel];
     adRequestModel.requestParameters[HyBidRequestParameter.locale] = [HyBidSettings sharedInstance].locale;
     
-//    Commenting out the following 3 parameters for now since the backend is not supporting those yet.
-//    adRequestModel.requestParameters[HyBidRequestParameter.versionOfOMSDKIntegration] = HYBID_OMSDK_VERSION;
-//    adRequestModel.requestParameters[HyBidRequestParameter.identifierOfOMSDKIntegration] = HYBID_OMSDK_IDENTIFIER;
+    adRequestModel.requestParameters[HyBidRequestParameter.versionOfOMSDKIntegration] = HYBID_OMSDK_VERSION;
+    adRequestModel.requestParameters[HyBidRequestParameter.identifierOfOMSDKIntegration] = HYBID_OMSDK_IDENTIFIER;
```
Comment (can-soykarafakili) [inline PubnativeLite/PubnativeLite/Ad Request/PNLiteAdFactory.m:53]: Are those fields supported in our API? Is it tested? During my implementation it was not supported and ad requests were failing.

File: PubnativeLite/Scripts/iqv.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        NEW=$(echo $i | cut -f2 -d,)
+        echo "Replace all occurences of $OLD within files to $NEW"
+        find . -type f \( -iname "*.storyboard" -or -iname "*.xib" -or -iname "*.swift" -or -iname "*.sh" -or -iname "*.xcworkspacedata" -or -iname "*.xcscheme" -or -iname "*.pbxproj" -or -iname "*.m" -or -iname "*.h" -or -iname "*.xib" -or -iname "*.plist" \) -print0 | xargs -0 sed -i '' -e 's/'"$OLD/$NEW"'/g'
+
+        for x in 1 2 3
+        do
+                echo "pass $x: Rename files from $OLD to $NEW"
+                find . -name "*$OLD*" -exec sh -c 'mv "$1" "$(echo "$1" | sed s/'"$OLD/$NEW"'/)"' _ {} \;
+        done
+done
```
Comment (can-soykarafakili) [inline PubnativeLite/Scripts/iqv.sh:28]: i am hoping this script is tested for running only on the IQV releases and not for our normal release.

Comment (eros902002) [reply PubnativeLite/Scripts/iqv.sh:28]: Yes. So far this part is being done manually. We need to add this to CI as another step but that's some another ticket.

File: PubnativeLite/PubnativeLite/Ad Request/PNLiteAdFactory.m (hunk 1 above)
Comment (eros902002) [reply PubnativeLite/PubnativeLite/Ad Request/PNLiteAdFactory.m:53]: Now they are. It was released on the ad server a few days ago.

**Review by eros902002 — APPROVED** [review]

**Pull request merged**